### DISCUSSION TITLE
Box shadow appearing overtop of text

### DISCUSSION THREAD

**Discussion by XMB-7** [chrisgurney/obsidian-note-toolbar#88] [General]
Currently when applied `border` or even `no-border`, the border always exists, so sometimes it looks like a strikethrough.
Maybe a 100% transparent border would help to prevent this.

![image](https://github.com/chrisgurney/obsidian-note-toolbar/assets/65625632/4e9303cf-ae3d-4db8-8862-be99e9e068a8)

**Comment by chrisgurney** [comment]
@XMB-7 Hello! Thanks for your comments.

1. It's not clear from your screenshot where your toolbar is, so I can troubleshoot this further. Can you provide a screenshot that includes the full toolbar? 

2. It might be helpful to see the configuration for the toolbar as well, that is if you're using a normal toolbar. (If it's a toolbar callout, I'd like to see the markdown for that.)

3. Are you using any themes here?

**Reply by XMB-7** [reply]
I just create a simple toolbar containing one item to test. 
<img src="https://github.com/chrisgurney/obsidian-note-toolbar/assets/65625632/ab8995c6-3877-4cea-a4bd-806b7d81ea80" width="300">

It seems to do nothing with any theme. I just tested in built-in sandbox vault with default theme in the dark mode. 
![recording](https://github.com/chrisgurney/obsidian-note-toolbar/assets/65625632/2400cdc7-47da-4551-b86e-f84e137b4441)

**Comment by chrisgurney** [comment]
@XMB-7 Thank you for the screenshots, much appreciated!

I can now reproduce this issue. Let me take a look and see what's going on here.

**Reply by chrisgurney** [reply]
@XMB-7 OK so here is the line in `styles.css` that adds this line across the text:

```
.cg-note-toolbar-bar-container {
   ...
    box-shadow: 0 0 0 1px var(--cg-nt-tbar-bg-color);
 ```
 
There's two workarounds you can try implementing until I can get to investigating further (so as not to break something else):
 
1. Create a CSS snippet to change the color.
2. Using the [Style Settings plugin](https://github.com/chrisgurney/obsidian-note-toolbar/wiki/Style-Settings-plugin-support) you can override that color, under `🎨 Colors > Toolbar: Background color` and make it transparent.

Let me know if you need help with either.

**Comment by chrisgurney** [comment]
Note to self: Two commits to look at when I'm able to get to this:

```
% git log -L 682,682:styles.css

commit 15a3b88afb9c87b3b0e84e94875a9a96a3d608d4
Author: Chris Gurney <chris@chrisgurney.ca>
Date:   Wed Apr 17 22:20:11 2024 -0400

    fix: missed Style Settings plugin color in container box-shadow

diff --git a/styles.css b/styles.css
--- a/styles.css
+++ b/styles.css
@@ -450,1 +450,1 @@
-    box-shadow: 0 0 0 1px var(--background-primary);
+    box-shadow: 0 0 0 1px var(--cg-nt-tbar-bg-color);

commit b0b7cd0d94401eada1666e8eb0b536a78f354a3f
Author: Chris Gurney <chris@chrisgurney.ca>
Date:   Tue Mar 26 06:07:10 2024 -0400

    fix: callout hover box-shadow past edges of sticky toolbar

diff --git a/styles.css b/styles.css
--- a/styles.css
+++ b/styles.css
@@ -303,0 +300,1 @@
+    box-shadow: 0 0 0 1px var(--background-primary);
```

**Comment by chrisgurney** [comment]
Closing discussion. If you're interested, follow issue #90 for updates.
